harold/plugins/http.py

```diff
@@ -46,14 +46,15 @@ def _authenticate_request(self, request):
             raise AuthenticationError
 
     def render_GET(self, request):
+        response = ""
         try:
             self._authenticate_request(request)
         except AuthenticationError:
             request.setResponseCode(403)
         else:
-            self._handle_request(request)
+            response = self._handle_request(request)
 
-        return ""
+        return response
```

Comment:
should this be `return response or ""` to deal with the other handlers which are just returning `None` by default?